### PR TITLE
Feature/snowflake tutorial ecc 4683

### PR DESCRIPTION
**Pull request by rpietzsch**

### CODE REVIEW

File: docs/build/snowflake-tutorial/22.2-Product-name-database.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-Product-name-database.png:1]: All snowflake screenshots should be prefixed with `snowlflake-` not `22.2.`.

All files should be lower cased only, (not `Product`).

File: docs/build/snowflake-tutorial/22.2-add-databasenew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-add-databasenew.png:1]: Why do we have 2 screens of the same content? Please remove one.

File: docs/build/snowflake-tutorial/22.2-add-projectnew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-add-projectnew.png:1]: same file again?

File: docs/build/snowflake-tutorial/22.2-add-schema.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-add-schema.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-add-value.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-add-value.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-advanced.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-advanced.png:1]: cropping not correct

naming to generic

File: docs/build/snowflake-tutorial/22.2-create-project.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-create-project.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-custome-pattern.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-custome-pattern.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-databas-name.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-databas-name.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-database-created.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-database-created.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-easynav-graph.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-easynav-graph.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-graph-target.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-graph-target.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-graph-uri.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-graph-uri.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-graph-urinew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-graph-urinew.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-graph.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-graph.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-jdbc-created.-new.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-jdbc-created.-new.png:1]: cropping not correct

file name wrong

remove all "new" from names

File: docs/build/snowflake-tutorial/22.2-jdbc-created.-newpng
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-jdbc-created.-newpng:1]: file name wrong

File: docs/build/snowflake-tutorial/22.2-jdbc-created.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-jdbc-created.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-mapping-rule.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-mapping-rule.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-mapping-rulenew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-mapping-rulenew.png:1]: remove all `new` from names

File: docs/build/snowflake-tutorial/22.2-organization-number.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-organization-number.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-output.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-output.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-project-name.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-project-name.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-suggestion-result.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-suggestion-result.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-suggestion-resultnew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-suggestion-resultnew.png:1]: remove all `new` from names

File: docs/build/snowflake-tutorial/22.2-suggestions.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-suggestions.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-suggestionsnew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-suggestionsnew.png:1]: remove all `new` from names

File: docs/build/snowflake-tutorial/22.2-target-entity-newpng.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-target-entity-newpng.png:1]: remove all `new` from names

File: docs/build/snowflake-tutorial/22.2-target-product.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-target-product.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-target-property.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-target-property.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-target-propertynew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-target-propertynew.png:1]: remove all `new` from names

File: docs/build/snowflake-tutorial/22.2-tick.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-tick.png:1]: cropping not correct

File: docs/build/snowflake-tutorial/22.2-ticknew.png
Comment (seebi) [inline docs/build/snowflake-tutorial/22.2-ticknew.png:1]: remove all `new` from names

**Review by seebi — CHANGES_REQUESTED** [review]
Please remove all old files
Please rename files accordingly

I can do a second review afterwards.

**Comment by rpietzsch** [comment]
@seebi please recheck.

- deleted duplicate / obsolete images
- images using style class (`bordered`) instead of of inline markup
- changed image filename prefix to `snowflake-`
- refined cropping of some images (removed borders, scrollbars, etc.)

**Review by seebi — APPROVED** [review]

**Review by seebi — APPROVED** [review]

**Pull request closed without merging**